website/modules/home-simplified/public/css/home-simplified.css

```
@@ -0,0 +1,558 @@
+@font-face {
+  font-family: 'Poppins';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)